### DISCUSSION TITLE
2022/03/Github-Actions-浅尝辄止/

### DISCUSSION THREAD

**Discussion by SouthFox-D** [SouthFox-D/SouthFox-D.github.io#62] [博客评论]
https://blog.southfox.me/2022/03/Github-Actions-浅尝辄止/

**Comment by ooorange777** [comment]
微软有钱嫖起来没有负罪感哈哈哈哈哈。
感谢南狐，用了GitHub Actions 这么久，终于在你这里看明白它是怎么运作的。

**Reply by SouthFox-D** [reply]
确实啊，毕竟微软（

能帮上忙就最好了~写的这么底层主要是当初用的是 `Travis` 的自动部署服务，没有别人写好的可以引用，而且经常改环境导致我去修……所以我在去年十二月才一气之下迁移过来的。

要是当初我就用 `GitHub Actions` 我估计也直接引用别人写好的实现了（

**Comment by so02iti01** [comment]
谢谢狐狸🦊，学生下次改进
